main.cpp

```diff
@@ -1,167 +1,251 @@
+
+10) click the [run] button.  Clear up any errors or warnings as best you can.
+ */
+
+// 2)
+void variableDeclarations()
+{
+    int numberOfSquirrels = 3;
+    int numberOfCats = 1;
+    int numberOfDogs = 2;
+
+    double magicNumber = 3.14;
+    double rootMeanSquare = 1.414;
+    double oddsOfRain = 0.5;
+
+    bool isRaining = false;
+    bool isSnowing = true;
+    bool isSunny = false;
+
+    float priceOfAdmission = 5.25f;
+    float priceOfLunch = 12.99f;
+    float priceOfCab = 20.75f;
+
+    char favouriteLetter = 'z';
+    char favouriteSymbol = '!';
+    char favouriteNumber = '4';
+
+    unsigned int planets = 8;
+    unsigned int provinces = 10;
+    unsigned int territories = 3;
+
+    ignoreUnused(numberOfSquirrels, numberOfCats, numberOfDogs, magicNumber, rootMeanSquare, oddsOfRain, isRaining, isSnowing, isSunny, priceOfAdmission, priceOfLunch, priceOfCab, favouriteLetter, favouriteNumber, favouriteSymbol, planets, provinces, territories);
+}
+
+/*
+ 10 functions
+ */
+
+/*
+ 1)
+ */
+int totalAnimals(int lions, int bears, double unicorns = 0.707)
+{
+    ignoreUnused(lions, bears, unicorns);
+    return {};
+}
+
+/*
+ 2)
+ */
+float costofOuting(float priceOfLunch, float priceOfAdmission, float priceOfCab)
+{
+    ignoreUnused(priceOfLunch, priceOfAdmission, priceOfCab);
+    return {};
+}
+
+/*
+ 3)
+ */
+bool itIsNiceOutside(int temperature, unsigned int humidity, int pop, unsigned int wind, bool sunny = true)
+{
+    ignoreUnused(temperature, humidity, pop, wind, sunny);
+    return {};
+}
+
+/*
+ 4)
+ */
+void printTripDetails(unsigned int numberOfpassengers = 3,
+                      double numberOfKilometers = 120,
+                      unsigned int month = 1,
+                      unsigned int day = 1,
+                      int year = 2024)
+{
+    ignoreUnused(numberOfpassengers, numberOfKilometers, month, day, year);
+}
+
+/*
+ 5)
+ */
+void recordExpenses(float priceOfLunch, float priceOfAdmission, float priceOfCab, int jobId = 1010, char clientId = 'c')
+{
+    ignoreUnused(priceOfLunch, priceOfAdmission, priceOfCab, jobId, clientId);
+}
+
+/*
+ 6)
+ */
+void addInitialsToForm(char firstInitial, char middleInitial, char lastInitial)
+{
+    ignoreUnused(firstInitial, middleInitial, lastInitial);
+}
+
+/*
+ 7)
+ */
+void setPosition(double x = 0.0, double y = 0.0, double z = 0.0)
+{
+    ignoreUnused(x, y, z);
+}
+
+/*
+ 8)
+ */
+unsigned int totalCandyBudget(unsigned int numberOfCandies,
+                              unsigned int numberOfPeople,
+                              unsigned int numberOfCandyPerPerson = 1,
+                              float costPerPiece = 0.42f)
+{
+    ignoreUnused(numberOfCandies, numberOfPeople, numberOfCandyPerPerson, costPerPiece);
+    return {};
+}
+
+/*
+ 9)
+ */
+char getLetterByPosition(unsigned int letterPositionInAlphabet)
+{
+    ignoreUnused(letterPositionInAlphabet);
+    return {};
+}
+
+/*
+ 10)
+ */
+bool trackIsArmed(int trackIndex = 1, bool absolute = true)
```

Comment:
This function name does not conform to the programming guideline:
```
Name Things what they are.
Name Actions based on what they do
```

The name you chose reflects a property (noun) and does not contain a verb, which would reflect an action.
Functions perform work, so their names must contain a verb which describes the work being performed.
Examples of function names that contain verbs:
- `computeAValue()`
- `estimateDistanceTraveled()`
- `performDiagnostics()`

- [ ] revise